packages/react-components/react-swatch-picker-preview/src/components/SwatchPicker/SwatchPicker.types.ts

```diff
@@ -1,17 +1,58 @@
+import * as React from 'react';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { SwatchPickerContextValue } from '../../contexts/swatchPicker';
 
 export type SwatchPickerSlots = {
   root: Slot<'div'>;
 };
 
+export type SwatchPickerOnSelectionChangeEvent = React.MouseEvent | React.KeyboardEvent | React.ChangeEvent;
```

Comment:
```suggestion
export type SwatchPickerOnSelectionChangeEvent = React.MouseEvent;
```

Please check, but AFAIR as `ColorSwatch` is a button it will be always `MouseEvent`

Reply:
yes, it's MouseEvent